portal/eproms/views.py

```diff
@@ -139,7 +139,19 @@ def privacy():
 def terms_and_conditions():
     """ terms-and-conditions of use page"""
     user = current_user()
-    terms = VersionedResource(app_text(Terms_ATMA.name_key()))
+    if user:
+        organization = user.first_top_organization()
+        role = None
+        for r in (ROLE.STAFF, ROLE.PATIENT):
+            if user.has_role(r) and not role:
+                role = r
+        if not all((role, organization)):
```

Comment:
Is this necessary? shouldn't `first_top_organization()` return something false-y enough to be `None`?

Reply:
Right, but if a `role` isn't found, then we don't want to send a value for `organization` either, and vice versa (because with `Terms_ATMA()`, if you have a null value for one param and a non-null value for the other, it throws an error; it only returns a value when either both params have values, or neither do)

Reply:
Actually, we COULD change `Terms_ATMA()` to never raise `ValueError`s, and just only return the org/role formatted terms if both params have values (otherwise return the non-formatted terms). Depends on if there's a good reason to raise those `ValueError`s...

Reply:
>change Terms_ATMA() to never raise ValueErrors, and just only return the org/role formatted terms if both params have values
Sorry, this is what I was suggesting, just not sure how good of an idea it was